10085-Pharo-does-not-work-on-network-drive-on-Windows-P10

tesonep opened the pull request:
Adding support for UNC Network paths.
They are required to correctly support window shares (SMB).

Fix #10085

tesonep committed:
Adding support for UNC Network paths.
They are required to correctly support window shares (SMB).

dionisiydk commented on src/FileSystem-Path/NetworkPath.class.st:
```diff
@@ -0,0 +1,68 @@
+A NetworkPath acts as an absolute path but with a different translation to string.
+"
+Class {
+	#name : #NetworkPath,
```
Maybe UNCNetworkPath would be a better name. The current one sounds too generic

dionisiydk reviewed (approved):
that's nice

tesonep committed:
- Renaming NetworkPath to UNCNetworkPath 
- Fixing transformation of ZnURL in MC to a correct absolute path.
- Extending the time of NoUnusedVariablesLeftTest

tesonep committed:
Handling correctly when the path is relative
